### ISSUE TITLE
Publish to maven central

### ISSUE THREAD

**Issue by jalexcole** [sbrunk/storch#77]
Hi, can you publish this to maven central?

**Comment by sbrunk** [comment]
Right, since we're on JavaCPP 1.5.10, we should be able to create a release. In the meantime, are you able to use the SNAPSHOT releases? https://s01.oss.sonatype.org/#nexus-search;quick~dev.storch

We never override older snapshots, as the version always contains the git sha.
